package.json

```diff
@@ -220,6 +222,16 @@
                 "title": "Disable Remote Debugging",
                 "category": "Azure App Service"
             },
+            {
+                "command": "appService.StartRemoteSsh",
+                "title": "Start Remote SSH",
```

Comment:
I feel like "SSH" already implies "remote"?

Reply:
Just Start SSH doesn't feel descriptive enough to me.  Start SSH into Web App?

Reply:
I mean... I'm already right clicking on the web app. We don't say "Start Web App" or "Delete Web App"